### DISCUSSION TITLE
NGINX reverse proxy with reiz NGINX forward proxy

### DISCUSSION THREAD

**Discussion by PowershellScripter** [reiz/nginx_proxy#16] [Q&A]
I have a backup application that I cannot use with my NGINX reverse proxy directly as it requires CONNECT requests. Am I able to use this forward proxy behind my NGINX reverse proxy so that I can connect to the backup server but not open additional ports or exposes what I dont need to?

Im nowhere near an expert on proxies so figured id ask much more knowledgeable people.

Thanks for responses in advance

**Comment by reiz** [comment]
Hi @PowershellScripter that's very less information you provided. A topographic landscape of your environment would be useful to give an educated response here. It really depends very much on your setup and your network configuration. But the Nginx forward proxy by default only supports port 80 and port 443 for SSL.
